universal-application-tool-0.0.1/app/repository/ApplicantRepository.java

```diff
@@ -34,6 +39,32 @@ public ApplicantRepository(EbeanConfig ebeanConfig, DatabaseExecutionContext exe
         () -> ebeanServer.find(Applicant.class).setId(id).findOneOrEmpty(), executionContext);
   }
 
+  public CompletionStage<ImmutableList<ProgramDefinition>> programsForApplicant(long id) {
```

Comment:
(blocking) can you add a docstring to describe which programs are for an applicant?

Reply:
Done!